stories/index.stories.js

```diff
@@ -99,49 +99,56 @@ storiesOf('Breadcrumb', module)
   )));
 
 storiesOf('Buttons', module)
-  .add('Primary Button', withInfo('')(() => (
+  .add('Primary Button', withInfo('Buttons set an action in motion. Instead of using “Yes” or “Okay”, it’s better to use verbs that explain what the triggered action will be. For example, “Create new”, “Discard draft”, etc. Use the primary button when it’s the main action on a page/table. Use the smaller one in case of limited space.')(() => (
     <div>
       <Button bsStyle="primary" onClick={action('clicked')}>{'Primary Button'}</Button>
       <Button bsSize="small" bsStyle="primary" onClick={action('clicked')}>{'Primary Small Button'}</Button>
     </div>
   )))
 
-  .add('Secondary Button', withInfo('')(() => (
+  .add('Default Button', withInfo('Default buttons are used for secondary actions. They’re usually paired with a primary button to provide an alternate choice.')(() => (
     <div>
-      <Button bsStyle="secondary" onClick={action('clicked')}>{'Secondary Button'}</Button>
-      <Button bsSize="small" bsStyle="secondary" onClick={action('clicked')}>{'Secondary Small Button'}</Button>
+      <Button bsStyle="default" onClick={action('clicked')}>{'Default Button'}</Button>
+      <Button bsSize="small" bsStyle="default" onClick={action('clicked')}>{'Default Small Button'}</Button>
     </div>
   )))
 
-  .add('Warning Button', withInfo('')(() => (
+  .add('Ghost Button', withInfo('Use ghost buttons to reduce visual clutter on the screen. Take care that the actions via a ghost button are not primary actions.')(() => (
+    <div>
+      <Button bsStyle="secondary" onClick={action('clicked')}>{'Ghost Button'}</Button>
```

Comment:
In Bootstrap v4 ghost buttons seem to correspond with outline buttons (I guess they're interchangeable): https://getbootstrap.com/docs/4.0/components/buttons/#outline-buttons. So https://github.com/metabrainz/design-system/pull/18 changes this to use those instead of the secondary style.